src/components/List/List.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai";
+
+import * as Enzyme from "enzyme";
+import * as React from "react";
+
+import List from "./List";
+import { ListTypes } from "./ListTypes";
+
+describe("Link With Underline", () => {
```

Comment:
Can you add at test for what happens if you pass in a bunch of `<span>`s into a `<List>` component?

Reply:
And maybe figure out if that should be tackled in DS, and how?